src/askContainer/webViews/simulateSkillWebview.ts

```diff
@@ -31,15 +28,14 @@ export class SimulateSkillWebview extends AbstractWebView {
     getHtmlForView(): string {
         Logger.debug(`Calling method: ${this.viewId}.getHtmlForView`);
 
-        const webview: vscode.Webview = this.getWebview() as vscode.Webview;
-        const simulateCss: vscode.Uri = webview.asWebviewUri(
+        const simulateCss: vscode.Uri = this.getWebview()?.asWebviewUri(
```

Comment:
why need a `?` here ? Seems could be deleted. Please apply to other places if it could be deleted

Reply:
Thanks Shen for pointing out this, deleted it in some places :)